src/biogeochem/CNVegCarbonFluxType.F90

```diff
@@ -397,7 +397,6 @@ subroutine Init(this, bounds, carbon_type)
 
     call this%InitAllocate ( bounds, carbon_type)
     call this%InitHistory ( bounds, carbon_type )
-    call this%InitCold (bounds )
```

Comment:
@wweider, I think this is a mistake, since you have changes to InitCold. This essentially get's rid of InitCold. So I don't think this is right.